src/CreatorTokenFactory.sol

```diff
@@ -30,7 +30,6 @@ contract CreatorTokenFactory {
   /// @param inflectionPrice The price at the point where the curve switches from quadratic to
```

Comment:
*arbitrary comment location*

I might recommend here that you don't modify this contract file directly, but instead create a new one with a new name as well. Can be `CreatorTokenFactoryV2` or `OpenCreatorTokenFactory` or whatever. You can of course copy the relevant code into that file.

The reason I'd suggest this is so you still have the source of the currently deployed version readily available on the main branch, with tests that execute etc... on the off chance you need to interact with the deployed version in the future.